libs/community/langchain_community/vectorstores/azuresearch.py

```diff
@@ -681,6 +1087,44 @@ def semantic_hybrid_search_with_score(
                 if score_threshold is None or reranker_score >= score_threshold
             ]
 
+    async def asemantic_hybrid_search_with_score(
+        self,
+        query: str,
+        k: int = 4,
+        score_type: Literal["score", "reranker_score"] = "score",
```

Comment:
I like your refactoring of the parameters to make them explicit. I guess you forgot this one though.

```suggestion
        score_type: Literal["score", "reranker_score"] = "score",
        *,
        score_threshold: Optional[float] = None
```